.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+    "presets": ["next/babel"]
```

Comment:
This file and code was necessary to get the jest tests to run properly. It seems to be a Next.js thing.